huaweicloud/services/apig/data_source_huaweicloud_apig_api_associated_throttling_policies.go

```diff
@@ -0,0 +1,392 @@
+package apig
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/go-uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+// @API APIG GET /v2/{project_id}/apigw/instances/{instance_id}/throttle-bindings/binded-throttles
+// @API APIG GET /v2/{project_id}/apigw/instances/{instance_id}/throttles/{throttle_id}/throttle-specials
+func DataSourceApiAssociatedThrottlingPolicies() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceApiAssociatedThrottlingPoliciesRead,
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"instance_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: `The ID of the dedicated instance to which the throttling policies belong.`,
+			},
+			"api_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: `The ID of the API bound to the throttling policy.`,
+			},
+			"policy_id": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: `The ID of the throttling policy.`,
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: `The name of the throttling policy.`,
+			},
+			"type": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: `The type of the throttling policy.`,
+			},
+			"env_name": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: `The name of the environment where the API is published.`,
+			},
+			"period_unit": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: `The time unit for limiting the number of API calls.`,
+			},
+			"policies": {
+				Type:        schema.TypeList,
+				Computed:    true,
+				Description: `All ACL policies that match the filter parameters.`,
```

Comment:
description is inconsistent with the document

Reply:
done.